Contribute.md

```diff
@@ -0,0 +1,32 @@
+# Contribute Guide
+
+this file contains about how to make a contribution
+
+## Whats Counts As A Good Logo
```

Comment:
These are good characteristics, but I don't want contributors to follow a strict format. For now, let's stick to just three criteria:
- Original and unique (that is, not copied from someone or too similar to an existing brand)
- Unused (not being used by another company or project)
- Beautiful (nice to look at)